src/js/visibleupgrades.js

```diff
@@ -22,7 +22,14 @@ var visibleUpgrades = function(GameState, UPGRADES, favico) {
 
         if(!meetsAllReqs) { return; }
 
-        _.each(item.levels, (level, i) => {
+        var levels = item.levels;
+        if(_.isFunction(item.levels)) {
+          var nextLevel = GameState.upgrade.getKey(itemName) || 0;
+          levels = {};
+          levels[nextLevel] = item.levels(nextLevel);
+        }
+
+        _.each(levels, (level, i) => {
```

Comment:
'arrow function syntax (=>)' is only available in ES6 (use esnext option).<br>'_' is not defined.